Packages/Application/Neos.NeosIo.Event/Configuration/NodeTypes.Page.yaml

```diff
@@ -0,0 +1,19 @@
+##############################################
+# Special default page for the event page tree
+##############################################
+
+'Neos.NeosIo.Event:Page':
+  superTypes:
+    'TYPO3.Neos.NodeTypes:Page': true
+  constraints:
+    nodeTypes:
+      'TYPO3.Neos.NodeTypes:Page': false
+      'Neos.NeosIo.Event:Event': false
+      'Neos.NeosIo.Event:Sponsor.Package': true
+      'Neos.NeosIo.Event:Sponsor.Partner': true
+  childNodes:
+    'keyvisual':
```

Comment:
something is missing here I guess - or this can be removed? :)

or do you want to unset the key visual? If yes, then I'd do "keyVisual: null"